code/modules/judaism/rabbi.dm

```diff
@@ -0,0 +1,103 @@
+/*
+PP Station 13 - Judaism Content
+Coded with love by woross
+Sprites hand drawn by KBB
+CONTENTS:
+-Judaism items
+-Rabbi Job
+*/
+
+/area/chapel/synagogue
+	name = "Synagogue"
+
+/obj/item/clothing/head/pp/kippah
+	name = "kippah"
+	desc = "A small Jewish hat."
+	icon_state = "kippah"
+	dynamic_hair_suffix = ""
+
+/obj/structure/sign/poster/contraband/israel
+	name = "Our Greatest Ally"
+	desc = "A poster with the flag of Nanotrasen's greatest ally. It looks quite damaged."
+	icon_state = "poster_israel"
+
+/obj/item/clothing/neck/necklace/dope/rabbi
+	desc = "Feels good to be rich."
+
+/obj/item/clothing/suit/toggle/lawyer/black/rabbi
+	name = "rabbi suit"
+	desc = "A formal suit."
+
+/obj/item/clothing/under/lawyer/black/rabbi
+	name = "formal clothes"
+	desc = "A formal shirt and pants."
+
+/obj/item/clothing/head/pp/jhat
+	name = "Jew hat"
+	desc = "A robust Jewish top hat."
+	icon_state = "jhat"
+
+
+/obj/effect/landmark/start/rabbi
+	name = "Rabbi"
+	icon = 'ppstation/icons/jew.dmi'
+	icon_state = "jewspawn"
+
+/datum/job/rabbi
+	title = "Rabbi"
+	flag = RABBI
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "G-d"
+	selection_color = "#dddddd"
+	access = list(ACCESS_SYNAGOGUE, ACCESS_MAINT_TUNNELS)
+	minimal_access = list(ACCESS_SYNAGOGUE, ACCESS_MAINT_TUNNELS)
+	outfit = /datum/outfit/job/rabbi
+	paycheck = PAYCHECK_COMMAND
+	paycheck_department = ACCOUNT_CIV
+
+/datum/job/rabbi/after_spawn(mob/living/carbon/human/H, mob/M)
+
+	var/lastname = pick(GLOB.last_names)
+	var/jewname = "Rabbi [lastname]stein"
+	H.fully_replace_character_name(H.real_name, jewname)
+	var/obj/item/organ/foreskin/F = H.getorgan(/obj/item/organ/foreskin)
+	if(F)
+		qdel(F)
+	if(H.gender == FEMALE)
+		H.gender = MALE
+		H.regenerate_icons()
+	var/datum/martial_art/S = H.mind.has_martialart("Catfighting")
+	if(S)
+		S.remove(H)
+
+
+/datum/outfit/job/rabbi
+	name = "Rabbi"
+	jobtype = /datum/job/rabbi
+	uniform =  /obj/item/clothing/under/lawyer/black/rabbi
+	shoes = /obj/item/clothing/shoes/laceup
+	suit = /obj/item/clothing/suit/toggle/lawyer/black/rabbi
+	head = /obj/item/clothing/head/pp/jhat
+	neck = /obj/item/clothing/neck/necklace/dope/rabbi
+
+/obj/item/book/manual/talmud
+	name = "The Talmud"
+	icon_state ="book1"
+	author = "Satan"
```

Comment:
"Leave your Anti-Semitism at the door"
https://tenor.com/view/senator-armstrong-armstrong-metal-gear-rising-mgr-speech-bubble-gif-25397532

Reply:
Yes, and?